.github/ISSUE_TEMPLATE/release.md

```diff
@@ -20,9 +20,16 @@ about: 'Start a new libp2p release.'
 - [ ] **Stage 0 - Finishing Touches**
     - [ ] Go through relevant libp2p repos looking for unreleased changes that should make it into the release. If you find any, cut releases.
     - [ ] Run `go get -u ./...` to see if there are any out-of-date deps that look important. If there are, bubble them. Try to avoid _directly_ updating indirect deps in go-libp2p's `go.mod` when possible.
+    - [ ] Performance Benchmarking
+      - [ ] Follow the instructions [on adding a new version to the libp2p performance benchmark tooling](https://github.com/libp2p/test-plans/tree/master/perf#adding-a-new-implementation-or-a-new-version). Instead of the final go-libp2p release commit, which obviously doesn't yet exist, use the most recent available commit. See https://github.com/libp2p/test-plans/pull/242 as an example.
+      - [ ] Run the benchmarks [via the GitHub _perf_ Action](https://github.com/libp2p/test-plans/tree/master/perf#running-via-github-action), wait for the action to push a commit to your https://github.com/libp2p/test-plans/ pull request and compare the performance of the upcoming release with the previous release [using the dashboard](https://github.com/libp2p/test-plans/tree/master/perf#running-via-github-action).
```

Comment:
This feels kind of heavy. Any chance we can add this to the uCI release workflow?

Reply:
Off the top of my head I can think of the following automation:

1. On creation of a new release in libp2p/go-libp2p
    1. Checkout https://github.com/libp2p/test-plans/
    2. Copy the most recent go-libp2p version folder in https://github.com/libp2p/test-plans/tree/master/perf/impl/go-libp2p
    3. Run `go get -u ...` to update to the to-be-tagged-as-release go-libp2p commit.
    4. Commit the changes, push them as a new branch to libp2p/test-plans and create a pull request.
    5. Trigger the `perf` GitHub Action on https://github.com/libp2p/test-plans/ with the above created branch.
    6. Post a comment on the pull request pointing users to `https://observablehq.com/@libp2p-workspace/performance-dashboard?branch=<LIBP2P-TEST-PANS-BRANCH-NAME>`

@galargh what do you think of the above? Any suggestions? Also would you be interested and have capacity to help?

@marten-seemann in case of a breaking change in go-libp2p you would still want to manually update the go-libp2p perf protocol implementation in libp2p/test-plans. Though this is not a blocker for any automation here. E.g. one could have the libp2p/test-plans pull request creation be automated and then only require the final touches to the pull request to be manual.

Reply:
Friendly ping @galargh. Any thoughts on the above?

Reply:
I quickly put together how something like this could look like - https://github.com/libp2p/test-plans/pull/256.